guides/data_science/machine_learning/Regression.md

```diff
@@ -0,0 +1,30 @@
+# Regression
+Towards **Machine Lerning** and **Data Science** *Regression* is the first topic to be ~~learned~~ *Mastered* . In the following we will try to introduce you about Regression and its basic types(there are many regression techniques but we cover few Important only):
```

Comment:
Learning not lerning
And why cover few important? Please cover all those available